pkg/asset/tls/tls.go

```diff
@@ -196,3 +190,30 @@ func GenerateSelfSignedCertificate(cfg *CertCfg) (*rsa.PrivateKey, *x509.Certifi
 	}
 	return key, crt, nil
 }
+
+// hasShortCertRotationEnabled returns true if ShortCertRotation featuregate is enabled.
+func hasShortCertRotationEnabled(installConfig *installconfig.InstallConfig) bool {
+	fgs := installConfig.Config.EnabledFeatureGates()
+	return fgs.Enabled(features.FeatureShortCertRotation)
+}
```

Comment:
Interesting. I browsed the enhancement to make sense of this and it says

```
This featuregate is not meant to be graduated to GA - its intended to be developer-only setting
```

So we will always use the feature gate to toggle the behavior on and off. It seems like we would have used env vars for this in the past, so now feature gates are an alternative.

Reply:
This feature gate affects other components - service-ca, kube-apiserver-operator, etcd-operator - so we need a centralised way for this to be enabled. Feature gate fits, although it is being usually used for a different purpose